hw/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Cubes</title>
+    <link rel="stylesheet" href="./src/css/style.css" type="text/css">
+</head>
+
+<body>
+    <section class="weatherWidget">
+        <div class="block first">
```

Comment:
BEM naming methodology is considered one of the best practices. (https://ru.bem.info/). Because the city tiles are all elements of the widget block, we able to name them accordingly, as a block and its elements:
```
<div class="weather-widget">
    <div class="weather-widget__block weather-widget__block_first"></div>
</div>
```
As you can see in the example above, elements are separated from the block by the "__" (double underscore). BEM methodology also allows for the convenient use of modifiers ("weather-widget__block_first") when we wish to use the same class on several tags at once and give them some individual characteristics in another class. The modifiers are separated with "_" (single underscore).